integration_test.go

```diff
@@ -0,0 +1,194 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"os"
+	"strings"
+	"testing"
+
+	"github.com/caarlos0/env/v11"
+	"github.com/rond-authz/rond/core"
+	"github.com/rond-authz/rond/internal/config"
+	"github.com/rond-authz/rond/internal/testutils"
+	"github.com/rond-authz/rond/openapi"
+	"github.com/sirupsen/logrus/hooks/test"
+	"github.com/stretchr/testify/require"
+	"gopkg.in/h2non/gock.v1"
+)
+
+func BenchmarkStartup(b *testing.B) {
+	b.StopTimer()
+
+	tmpdir, err := os.MkdirTemp("", "console-")
```

Comment:
what is console? I don't know anything about this 😆 

```suggestion
	tmpdir, err := os.MkdirTemp("", "rond-bench-")
```

Reply:
LOL, ops it's habit 🙄